content/guides/user-guides/multi-cloud/storage-service-plan/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Storage Service Plan
+description: "User guide for Storage Service Plan(SSP) Management."
+weight: 10
+disableToc: false
+tags: ["user guide", "storage service plan", "multi-cloud"]
+---
+
+## Introduction to Storage Service Plan(SSP) management
+SODA is aimed at addressing the storage integration challenges of both the Cloud Native environment and traditional IT environment. SODA multi-cloud allows service providers(admin) or users to manage distributed cloud and on-premise environments on a single platform.
+
+
+SSP management allows service providers to abstract actual cloud or on-premise environments(backends) from their users.
+In other words, SSP is a policy for the users. A service provider can decide and create different types of SSP policy for different tenants of users.
+
+## Getting Started!
+This is a guide that shows how to install, configure, and use SSP management feature in a simple SODA setup.
+By following this guide you will be able to install SODA multi-cloud with creating backends and buckets, uploading and archiving objects.
+
+## Installing SODA multi-cloud
+
+### Download SODA-installer code
+
+```cassandraql
+git clone https://github.com/sodafoundation/installer.git
+cd installer/ansible
+# Checkout the latest release, give master by default or specify the branch
+git checkout master
+```
+
+### How to enable multi-cloud installation with SSP feature
+
+Install docker container to run multi-cloud in container environment.
+
+Update the file :
+
+a. group_vars/gelato.yml to install multi-cloud.
+```cassandraql
+# repository branch
+gelato_repo_branch: master
+```
+
+b. group_vars/common.yml to enable SSP feature
+
+```
+enable_storage_service_plans: true
+```
+Detailed instruction is in this link
+
+- [SODA Local Cluster Installation through Ansible On Ubuntu](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible) (Recommended)
+
+#### Export required SODA  environment variables
+```
+export HOST_IP={your_real_host_ip}
+export OPENSDS_ENDPOINT=http://{{ apiserver_cluster_ip }}:50040
+export OPENSDS_AUTH_STRATEGY=keystone
+export OS_AUTH_URL=http://{{ authchecker_cluster_ip }}/identity
+```
+#### Run SODA multi-cloud in docker container
+```
+docker-compose up -d
```

Comment:
This is separate from the installation through SODA installer, right? This is mostly for developers